pkg/networkmanager/dialogs-common.jsx

```diff
@@ -66,7 +66,7 @@ export const MacMenu = ({ idPrefix, model, mac, setMAC }) => {
         });
         optionsMapInit.push(
             { toString: () => _("Permanent"), value: "permanent" },
-            { toString: () => _("Perserve"), value: "preserve" },
+            { toString: () => _("Preserve"), value: "preserve" },
```

Comment:
Collides with #16727, so this will need rebasing.

Reply:
Or we can just close #16727 if this has chance to land reasonably quickly as well

Reply:
I landed #16727, and git+github are clever enough to rebase over it.